.github/workflows/node.js.yml

```
@@ -3,6 +3,7 @@ env:
   CI: false
   GITHUB_USERNAME: ${{ github.repository_owner }}
   REACT_APP_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Use the generated personal access token and add it to repository secrets with the name OPEN_SOURCE_TOKEN
+  USE_GITHUB_DATA: ${{ secrets.USE_GITHUB_DATA }}
```

why `${{ secrets.USE_GITHUB_DATA }}`?
```suggestion
  USE_GITHUB_DATA: "true"
```

---

Wouldn't this cause failing builds every time if the other variables aren't set?

---

> Wouldn't this cause failing builds every time if the other variables aren't set?

Well, when running Github Actions, Github gives us a token to access their API and we use that for fetching details, the line
```
REACT_APP_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
means, it uses the one provided by Github itself, that comment near to it doesn't make sense though.

So, it means, the user doesn't need to do anything for Github Actions to work and this won't give failing builds.

---

Ah that does make sense, I thought it would fail requests if we don't provide other vars.